lib/classes/Swift/Transport/SendmailTransport.php

```diff
@@ -102,6 +102,7 @@ public function send(Swift_Mime_Message $message, &$failedRecipients = null)
         $failedRecipients = (array) $failedRecipients;
         $command = $this->getCommand();
         $buffer = $this->getBuffer();
+        $count = 0;
```

Comment:
Note for reviewers: variable may not have been defined and initialized later in the code.